src/transformers/models/longformer/modeling_longformer.py

```diff
@@ -404,12 +404,16 @@ def _compute_global_attention_mask(input_ids, sep_token_id, before_sep_token=Tru
     # bool attention mask with True in locations of global attention
     attention_mask = torch.arange(input_ids.shape[1], device=input_ids.device)
     if before_sep_token is True:
-        attention_mask = (attention_mask.expand_as(input_ids) < question_end_index).to(torch.uint8)
+        attention_mask = (attention_mask.expand_as(input_ids) < question_end_index).to(
+            torch.bool
+        )  # previously `dtype=torch.uint8`, cf pytorch 1.2.0 compatibility
```

Comment:
I don't think we need the comment, but if we keep it, let's put it above the line of code as it screws up formatting.